django_extras/db/models/fields/__init__.py

```diff
@@ -71,7 +71,7 @@ def get_db_prep_save(self, value, connection):
         value = self.to_python(value)
         if value is not None:
             value = value._amount
-        return connection.ops.value_to_db_decimal(value, self.max_digits, self.decimal_places)
+        return connection.ops.adapt_decimalfield_value(value, self.max_digits, self.decimal_places)
```

Comment:
Is this change backwards compatible with Django < 2? 

This might require a conditional to handle previous Django releases

Reply:
Unfortunately, I don't know, as everything I'm doing now is Django 2+. But it seems that 'value_to_db_decimal' disappeared in Django 1.9